tools/python/test/test_numpy_returns.py

```diff
@@ -10,36 +10,40 @@
 image_path = "examples/faces/Tom_Cruise_avp_2014_4.jpg"
 shape_path = "tools/python/test/shape.pkl"
 
+
 def get_test_image_and_shape():
     img = dlib.load_rgb_image(image_path)
     shape = utils.load_pickled_compatible(shape_path)
     return img, shape
- 
+
+
 def get_test_face_chips():
     rgb_img, shape = get_test_image_and_shape()
     shapes = dlib.full_object_detections()
     shapes.append(shape)
     return dlib.get_face_chips(rgb_img, shapes)
 
+
 def get_test_face_chip():
     rgb_img, shape = get_test_image_and_shape()
     return dlib.get_face_chip(rgb_img, shape)
 
+
 @pytest.mark.skipif(not utils.is_numpy_installed(), reason="requires numpy")
 def test_partition_pixels():
-    truth = (102, 159, 181);
+    truth = (102, 159, 181)
     img, shape = get_test_image_and_shape()
 
-    assert(dlib.partition_pixels(img) == truth[0])
-    assert(dlib.partition_pixels(img,3) == truth)
+    assert dlib.partition_pixels(img) == truth[0]
+    assert dlib.partition_pixels(img, 3) == truth
 
     # Call all these versions of this mainly to make sure binding to
     # various image types works.
-    assert(dlib.partition_pixels(img[:,:,0]) == 125)
-    assert(dlib.partition_pixels(img[:,:,0].astype('float32')) == 125)
-    assert(dlib.partition_pixels(img[:,:,0].astype('float64')) == 125)
-    assert(dlib.partition_pixels(img[:,:,0].astype('uint16')) == 125)
-    assert(dlib.partition_pixels(img[:,:,0].astype('uint32')) == 125)
+    assert dlib.partition_pixels(img[:, :, 0].astype("uint8")) == 125
```

Comment:
I don't understand why this doesn't work anymore… When I print the type of `img[:, :, 0]`, it is an `ndarray` of `numpy.uint8`.